frontend/src/i18n/locales/en.yaml

```diff
@@ -390,6 +390,15 @@ manage:
         confirm: Delete Video?
         cannot-be-undone: This action <strong>cannot</strong> be undone!
         failed: Deleting the video failed.
+        pending: Deletion is pending.
+        tooltip:
+          failed: >
+            Deleting this video in Opencast failed. It should be getting removed 
+            by the responsible administrator soon.
```

Comment:
The second sentence is a promise we can't keep :P I mean yes, it should be. But we probably shouldn't say it like that. And yes you already use "should"... but still, maybe just remove that part? Also, the first part sounds so definitive when in fact we are just guessing. 

Maybe this and the `pending` string below should actually mention the time when it was deleted? Or just this one actually?